inlong-dataproxy/dataproxy-docker/dataproxy-docker.sh

```diff
@@ -17,7 +17,7 @@
 #
 
 file_path=$(cd "$(dirname "$0")"/../;pwd)
-local_ip=$(ifconfig | grep inet | grep -v inet6 | grep -v "127.0.0.1" | awk '{print $2}' | grep -v "30.*")
+local_ip=$(ifconfig | grep inet | grep -v inet6 | grep -v "127.0.0.1" | awk '{print $2}' | grep -v "^30.*")
 # config
```

Comment:
It is necessary to supplement the IP filling rules here for the convenience of subsequent people, because if an environment uses the address of 30.xxx.xxx.xxx, they will have similar errors

Reply:
> It is necessary to supplement the IP filling rules here for the convenience of subsequent people, because if an environment uses the address of 30.xxx.xxx.xxx, they will have similar errors

Ok, I'll add some necessary explanatory notes.